modules/nw-multinetwork-sriov.adoc

```diff
@@ -40,247 +62,386 @@ and device ID 0x1015
+  priority: 99 <5>
+  mtu: 9000 <6>
+  numVfs: 16 <7>
+  nicSelector: <8>
+    vendor: "15b3" <8>
+    deviceID: "" <9>
+    pfName: ["eno3", "eno4"] <10>
+    rootDevices: ["0000:02:00.0", "0000:02:00.1"] <11>
+  deviceType: netdevice <12>
+  isRdma: false <13>
 ----
-+
-<1> `k8s.v1.cni.cncf.io/resourceName` annotation is set to `openshift.io/sriov`.
-<2> `type` is set to `sriov`.
+<1> The name of the CR.
+<2> The namespace of the CR, this must be in the same namespace of operator.
+<3> The resource name of SR-IOV device plug-in. Prefix `openshift.io/` will be
+added when it's referred in pod spec. It's allowed to create multiple CRs of
+`SriovNetworkNodePolicy` for one resource name.
+<4> The node selector to select which node to be configured. User can choose to
+label the nodes manually or with tools like Kubernetes Node Feature Discovery.
+Only SR-IOV network devices on selected nodes will be configured. And the SR-IOV
+CNI plug-in and device plug-in will be only deployed on selected nodes.
+<5> The priority of the policy,  the larger number gets lower priority.
+Range from 0 to 99.
+<6> The MTU of the virtual functions. Range from 1 to 9000. Leave it blank if
+you don't need to change the MTU.
+<7> The number of the virtual functions is to be created for each SR-IOV
+physical network device.
+<8> The NIC selector selects the NIC to be configured. You don't have to specify
+all the fields. However it is recommended to select the NICs as specifically as
+possible, in order to minimize the possibility of overlapping cross policies.
+And if you specify both `pfName` and `rootDevices` at the same time, which is
+recommended, please make sure they point to the identical devices.
+<9> The vendor hex code of SR-IoV device. Allowed value "8086", "15b3".
+<10> The device hex code of SR-IoV device. Allowed value "158b", "1015", "1017".
+<11> The names of SR-IoV physical function.
+<12> The PCI addresses of SR-IoV physical function. 
+<13> The driver type of the virtual functions. Allowed value "netdevice",
+"vfio-pci". Defaults to "netdevice".
+<14> The RDMA mode. Defaults to false. In this release, only RoCE mode are
+supported, and only for Mellanox NICs.
 
-. Run the following command to create the `sriov-conf` CR:
 +
+[NOTE]
+=====
+When `RDMA` flag is configured to true, it doesn't prevent user from using RDMA
+enabled VF as a normal network device. A device can be used in either mode.
+=====
+
+== Configuring SR-IOV networks
+
+The SR-IOV Network Operator also introduces a CRD `SriovNetwork` for creating
+the Custom Resource of `NetworkAttachmentDefinition` of SR-IOV CNI plug-in. When
+you create a CR of `SriovNetwork`, the operator will created a CR of
+`NetworkAttachmentDefinition` accordingly.
+
+You should never have to modify this CRD. Instructions for creating or 
+modifying a CR are provided in this documentation as appropriate.
+
+[NOTE]
+=====
+You shall not modify or delete a Custom Resource of `SriovNetwork`, when it has
```

Comment:
Is there a simple command to query which `SriovNetwork` are in use by a particular pod?

Reply:
I don't think there is any.